timed/permissions.py

```diff
@@ -28,6 +28,16 @@ def has_object_permission(self, request, view, obj):
         return self.has_permission(request, view)
 
 
+class IsNotDelete(BasePermission):
+    """Allows only delete method."""
```

Comment:
comment needs to be adjusted.... although in this case the class name is quiet clear so might just also remove it.